### PR TITLE
💡Add notes on testing, translations, merging and starting Refrisk

### PR DESCRIPTION
**Pull request by shivekkhurana**
# Summary
My notes on test processes, merging PRs and translations. Might be helpful for new team mates.

Insights provided by @churik and @flexsurfer. Compiled by yours truly.

status: ready

### CODE REVIEW

File: doc/codebase-structure-and-guidelines.md [hunk 1]
```diff
@@ -55,13 +62,15 @@ Rationale:
 
 These guidelines make db.cljs namespaces the place to go when making changes to the db layout and minimize breaking changes when adding/refactoring features
 
-# Subscriptions
+## Reframe
```
Comment (flexsurfer) [inline doc/codebase-structure-and-guidelines.md]:
```suggestion
## re-frame
```

File: doc/codebase-structure-and-guidelines.md [hunk 2]
```diff
@@ -78,3 +87,52 @@ These guidelines make db.cljs namespaces the place to go when making changes to
...
+    - No issues - label 'TESTED-OK' and then you can merge it.
+    - If issues - label 'TESTED-ISSUES' and comment from QA with bugs.
+    - After fix and/or discussion the process is repeated.
+
+  - If manual QA is not needed but all tests don't pass, you can ping @churik or @Serhy to confirm that failed E2E tests are not unrelated. 
+
+## Enabling debug logs
+Calls to `log/debug` will not be printed to the console by default. It can be enabled under "Advanced settings" in the app:
+
+![Enable Debug Logs](https://media.discordapp.net/attachments/626738033701683200/778912893105995786/unknown.png)
```
Comment (flexsurfer) [inline doc/codebase-structure-and-guidelines.md]: not sure this link will be always available

File: doc/codebase-structure-and-guidelines.md [hunk 3]
```diff
@@ -78,3 +87,52 @@ These guidelines make db.cljs namespaces the place to go when making changes to
...
+
+## Translations
+The app relies on system locale to select a language from the [list of supported languages](https://github.com/status-im/status-react/blob/bda73867471cf2bb8a68b1cc27c9f94b92d9a58b/src/status_im/i18n_resources.cljs#L9). It falls back to English in cash the system locale is not supported.
+
+We use Lokalise App to manage [translations](https://translate.status.im/). In case you need to add/remove a key to translations, you only need to change `en.json`. Missing keys fallback to `en.json`. The actual translations will be added by Lokalise.
+
+## Refrisk
```
Comment (flexsurfer) [inline doc/codebase-structure-and-guidelines.md]:
```suggestion
## re-frisk
```

File: doc/codebase-structure-and-guidelines.md [hunk 4]
```diff
@@ -78,3 +87,52 @@ These guidelines make db.cljs namespaces the place to go when making changes to
...
+Refrisk is a state visualization tool written by our very own Andrey (@flexsurfer). To start Refrisk, execute the following command:
```
Comment (flexsurfer) [inline doc/codebase-structure-and-guidelines.md]:
```suggestion
re-frisk is a state visualization tool written by our very own Andrey (@flexsurfer). To start re-frisk, execute the following command:
```

File: doc/codebase-structure-and-guidelines.md [hunk 5]
```diff
@@ -78,3 +87,52 @@ These guidelines make db.cljs namespaces the place to go when making changes to
...
+```bash
+$ yarn shadow-cljs run re-frisk-remote.core/start
+```
```
Comment (flexsurfer) [inline doc/codebase-structure-and-guidelines.md:119]: or 
```bash
$ make run-re-frisk
```

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: doc/codebase-structure-and-guidelines.md (hunk 2 above)
Comment (shivekkhurana) [reply doc/codebase-structure-and-guidelines.md]: Uploaded image to this repo and created a local link.

**Review by flexsurfer — APPROVED** [review]

**Review by churik — APPROVED** [review]
👍

**Pull request merged**